go-controller/pkg/ovn/controller/unidling/ovn_event.go

```diff
@@ -1,103 +1,32 @@
 package unidling
 
 import (
-	"encoding/json"
-	"errors"
-
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/sbdb"
```

Comment:
we already have https://github.com/ovn-org/ovn-kubernetes/pull/2262 for unidling refactor

Reply:
Damn I duplicated some work here then... Lets get that merged before this and I'll rip out my commits in a rebase

Reply:
@astoycos 2262 is merged; rebase?

Reply:
Will do thanks @dcbw I need to rip out some commits from here as well so I'll get it done later today

Reply:
Done